registry.go

```diff
@@ -38,6 +41,13 @@ type RegistryService interface {
 	GetSubscription(context.Context) (*RegistrySubscription, *Response, error)
 	UpdateSubscription(context.Context, *RegistrySubscriptionUpdateRequest) (*RegistrySubscription, *Response, error)
 	ValidateName(context.Context, *RegistryValidateNameRequest) (*Response, error)
+
+	// Multi-registry Open Beta API methods
+	GetBeta(context.Context, string) (*Registry, *Response, error)
+	ListBeta(context.Context) ([]*Registry, *Response, error)
+	CreateBeta(context.Context, *RegistriesCreateRequest) (*Registry, *Response, error)
+	DeleteBeta(context.Context, string) (*Response, error)
+	DockerCredentialsBeta(context.Context, string, *RegistryDockerCredentialsRequest) (*DockerCredentials, *Response, error)
```

Comment:
I don't think we should be creating temporary methods for betas. If we need to distinguish these as being in beta, we can do so with comments. The current approach will force us to make breaking changes for users and downstream components or carry these forward indefinitely. I think the better approach might be to create a separate `RegistriesService` much like how the new API itself is under `/v2/registries`.